src/script/page/MainContent/panels/preferences/avPreferences/DeviceSelect.tsx

```diff
@@ -59,15 +59,19 @@ const DeviceSelect: React.FC<DeviceSelectProps> = ({
       <div className="preferences-option-icon preferences-av-select-icon">
         <DeviceIcon />
       </div>
+
       <div css={{width: '550px'}}>
         <Select
           id={uieName}
-          onChange={onChange}
+          onChange={device => {
+            const deviceOption = device as Option;
+            onChange(deviceOption.value as string);
+          }}
```

Comment:
im still not clear why this is necessary, Select expects its onChange value to be a type Option or Option[]
```suggestion
          onChange={(device:Option) => onChange(device.value.toString())}
```

Reply:
<img width="860" alt="image" src="https://user-images.githubusercontent.com/13432884/181498404-05590e0d-8f3a-4deb-b820-ed9201d8562b.png">

Reply:
https://github.com/wireapp/wire-webapp/pull/13398/files#r932137696

Reply:
Fixed.